clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -3318,6 +3353,20 @@ llvm::Constant *CodeGenFunction::EmitCheckTypeDescriptor(QualType T) {
       DiagnosticsEngine::ak_qualtype, (intptr_t)T.getAsOpaquePtr(), StringRef(),
       StringRef(), std::nullopt, Buffer, std::nullopt);
 
+  if (IsBitInt) {
+    // The Structure is: 0 to end the string, 32 bit unsigned integer in target
+    // endianness, zero.
+    char s[6] = {'\0', '\0', '\0', '\0', '\0', '\0'};
+    const auto *EIT = T->getAs<BitIntType>();
```

Comment:
```suggestion
    const auto *EIT = T->castAs<BitIntType>();
```
We don't expect it to ever return a null pointer, so we might as well cast instead.

Reply:
Agreed. We already know the type is some sort of BitIntType.